profiler/profiler.go

```diff
@@ -259,6 +259,13 @@ func (p *profiler) collect(ticker <-chan time.Time) {
 			seq:   p.seq,
 			host:  p.cfg.hostname,
 			start: now(),
+			extraTags: []string{
+				// go_execution_trace_enabled indicates whether execution
+				// tracing is enabled, to distinguish between missing a trace
+				// because we don't collect them every profiling cycle from
+				// missing a trace because the feature isn't turned on.
+				fmt.Sprintf("go_execution_trace_enabled:%v", p.cfg.traceConfig.Enabled),
```

Comment:
Do we expect users to use this tag? If not, maybe we should use `_dd.profiler.go_execution_trace_enabled` ? This was proposed in Ivo's internal doc. A `_dd.profiler.` prefix would also make it easier to hide these kind of tags in the UI (if we want to hide them).

(This is just a question, I'm not asking you to change the tag right away)